package haptix.comm.msgs;

import "hxVector3.proto";

message hxContactPoint
{
  required string link1        = 1;
  required string link2        = 2;
  required hxVector3 point     = 3;
  required hxVector3 normal    = 4;
  required hxVector3 tangent1  = 5;
  required hxVector3 tangent2  = 6;
  required float distance      = 7;
  required hxVector3 velocity  = 8;
  required hxVector3 force     = 9;
}
